release notes/v0.43.0.md

```diff
@@ -0,0 +1,125 @@
+
+The `k6/http` module has a new `asyncRequest` function that takes the same arguments as [`http.request()`](https://k6.io/docs/javascript-api/k6-http/request/), but returns a `Promise` that when `await`ed will be resolved with a `Response` object. This gives you more control over the script execution, as potentially the most time-consuming calls—making HTTP requests—will no longer block the thread of execution.
+
+See this example issuing a POST request:
+
+```javascript
+import http from 'k6/http';
+
+export default async function () {
+  const resPromise = http.asyncRequest(
+    'POST', 'https://httpbin.test.k6.io/post', { name: 'Bert' });
+  // Do something else here, make other requests, etc.
+  // Then when you're ready to use the response:
+  const resp = await resPromise;
+  console.log(resp.json().form.name); // Bert
+}
+```
+
+This is one of the first steps towards migrating our APIs to be asynchronous, and similar changes can be expected in the future.
+
+You can read more about `asyncRequest` in the [documentation](https://k6.io/docs/javascript-api/k6-http/asyncrequest/).
+
+
+### Enhancements and UX improvements
+
+- [#2754](https://github.com/grafana/k6/pull/2754), [#2805](https://github.com/grafana/k6/pull/2805) The output of the `k6 version` command has been enhanced to also show the version of all extensions built into the k6 binary produced by [xk6](https://github.com/grafana/xk6). Thanks, @HarrisChu!
+- [#2800](https://github.com/grafana/k6/pull/2800) Improved handling of the Ctrl+C signal to gracefully abort the test during VU initialization.
+- [#2803](https://github.com/grafana/k6/pull/2803) Ensure the REST API server is shut down after the test ends.
+- [#2867](https://github.com/grafana/k6/pull/2867) Added the ability to display test run details and logs from k6 Cloud.
+- [#2890](https://github.com/grafana/k6/pull/2890) Added a method for JavaScript modules to lookup environment variables without directly accessing the `os` package.
+
+
+## Bug fixes
+
+- [#2829](https://github.com/grafana/k6/pull/2829) The `csv` output now correctly shows `vu` and `iter` system tags. This fixes a regression introduced in v0.40.0. Thanks, @leonyork!
+- [#2851](https://github.com/grafana/k6/pull/2851) Calling `k6/execution.test.abort()` within a `group()` now correctly exits the k6 process with code `108`. Thanks for reporting this, @pomeh!
+- [#2896](https://github.com/grafana/k6/pull/2896) Fixed a panic in `k6/ws` when using `Socket.setInterval()` with values between 0 and 1.
+- [#2903](https://github.com/grafana/k6/pull/2903) JS modules are now cached by absolute, and not relative paths.
+
+  This fixes a regression introduced in v0.42.0 ([#2773](https://github.com/grafana/k6/pull/2773)), and has the side-effect of changing how modules are cached internally, which may cause issues for scripts that relied on the previous buggy behavior. See the PR [#2903](https://github.com/grafana/k6/pull/2903) for details.
```

Comment:
```suggestion
- [#2903](https://github.com/grafana/k6/pull/2903) Fixed a [regresion](https://github.com/grafana/k6/issues/2902) introduced in v0.42.0 where k6 will load the wrong module if the same import specifier has already been loaded, but they are pointing to different absolute paths, based on the files they are imported from.
```

This is very unlikely to be hit by many people so having a long description seems not as useful.

I also decided to word it for what will be observed then what is the root cause.